wolfcrypt/src/ecc.c

```diff
@@ -13220,7 +13282,11 @@ int wc_ecc_decrypt(ecc_key* privKey, ecc_key* pubKey, const byte* msg,
              #endif
                 ret = wc_AesInit(aes, NULL, INVALID_DEVID);
                 if (ret == 0) {
-                    ret = wc_AesSetKey(aes, encKey, encKeySz, encIv,
+                    byte ctr_iv[16];
+                    /* Make a 16 byte IV from the 12 bytes passed in. */
```

Comment:
Is this always true for all ECIES modes and AES CTR... 12 bytes passed in?

Reply:
Made configurable.
Use case was Google Pay.